src/chapters/04/main.tex

```diff
@@ -397,9 +397,16 @@ \section{Solving with R}\label{sec:solving-with-R}
 
 \section{Research}\label{sec:research}
 
-TBA
-
+There are a number of further areas related to the study of as well as the use
+of differential equations. Topics omitted here include the actual solution
+approaches which in this chapter are taken
+care of using open source software. Chapters 9, 14 and 16
+of~\parencite{stewart2009probability} provide a good introduction to some of
```

Comment:
are you sure you meant this book?

Reply:
No the Calculus book is also by a Stewart.

Reply:
This is fixed.